README.md

```diff
@@ -173,6 +174,9 @@ Monitor and record performance metrics: CPU, memory, network usage, and FPS (gam
 ### resultsBucket
 The name of a Google Cloud Storage bucket where raw test results will be stored.
 
+### keepFilePath
+Keeps the full path of downloaded files from a Google Cloud Storage bucket. Required when file names are not unique. Disabled by default.
```

Comment:
what file names would not be unique? how does one know that one should enable this?

Reply:
As an example. When the toggle is disabled downloaded files are stored at the root of device.
`results/2019-09-20-13-57-45.1454_nGys/shard_0/NexusLowRes-28-en-portrait/file.png`

But when it is enabled, then the full path of the file kept.
`results/2019-09-20-13-57-45.1454_nGys/shard_0/NexusLowRes-28-en-portrait/{package name}/file.png`